plugins/generic/externalFeed/ExternalFeedPlugin.inc.php

```diff
@@ -1,366 +1,328 @@
-<?php
```

Comment:
Did the line endings change format from UNIX to DOS or something here? I'm not sure why the entire file is rewritten.

Reply:
Did you look into this?

Reply:
yes. but to be honest, I am not sure what's happening here. any hint?

Reply:
It looks like your editor converted the file from UNIX-like line endings to DOS line endings. You can use `dos2unix` to convert it back, but also make sure your editor is set up for UNIX endings so it doesn't happen again.